packages/animations/test/open_container_test.dart

```diff
@@ -1486,7 +1486,7 @@ void main() {
       (WidgetTester tester) async {
     bool hasClosed = false;
     final Widget openContainer = OpenContainer(
-      onClosed: () {
+      onClosed: (dynamic _) {
```

Comment:
This is technically a breaking change for any implementation of `OpenContainer` since the function signature has changed. Do we just mention this in the changelog?

Reply:
Since there is no other way to make the parameter optional, I think just mentioning it in the changelog would be the way to go.

Reply:
That makes sense! Could you add an entry to [the changelog](https://github.com/flutter/packages/blob/master/packages/animations/CHANGELOG.md) with a migration guide?

Reply:
@shihaohong I had another look at it and I feel that the changes should not break existing codebase which is evident by the fact that not much of the changes are made to the `open_container_test.dart` file and still it works just fine. Still, let me know if there are any specific cases where it might break.

Reply:
I was mostly worried about existing users of the `onClosed` parameter, but as you pointed out, it was not available in 1.0.0+5 so it should be okay!